docs/playbooks/lite-assist-quickburn.md

```diff
@@ -0,0 +1,125 @@
+# MAS Assist Service on DevIT Quickburn
+
```

Comment:
Opening sentence to say this will be used to deploy CP4D and watson discovery on this cluster

Reply:
updated